src/h5web/vis-packs/core/heatmap/ColorBar.module.css

```diff
@@ -21,9 +21,15 @@
     '. axis .';
 }
 
-.gradient {
+.gradientBar {
   grid-area: gradient;
+  display: flex;
   border: 1px solid var(--black);
+}
+
+.gradient {
+  composes: no-invert from global;
```

Comment:
I had move the gradient to a child element so as to not invert the border of the gradient bar.